src/managers/GuildManager.js

```diff
@@ -211,6 +211,25 @@ class GuildManager extends BaseManager {
         }, reject),
     );
   }
+
+  /**
+   * Obtains a guild from Discord, or the guild cache if it's already available.
+   * @param {Snowflake} id ID of the guild
+   * @param {boolean} [cache=true] Whether to cache the new guild object if it isn't already
+   * @returns {Promise<Channel>}
```

Comment:
```suggestion
   * @returns {Promise<Guild>}
```
Copy and paste mistake? :)